lib/src/update_version.dart

```diff
@@ -13,10 +14,28 @@ abstract class UpdateVersion extends Command {
 
   Future run() async {
     final dir = Directory(globalResults['pubspec-dir']);
-    final pubSpec = await PubSpec.load(dir);
-    final version = nextVersion(pubSpec.version);
-    await pubSpec.copy(version: version).save(dir);
-    console.log(version.toString());
+    final File pubSpec = File(p.join(dir.path, "pubspec.yaml"));
+
+    Version oldVersion = VersionExtractor.extractVersion(pubSpec);
+    Version newVersion = nextVersion(oldVersion);
+    console.log(newVersion.toString());
+
+    _writeNewPubSpec(pubSpec, oldVersion, newVersion);
+  }
+
+  Future _writeNewPubSpec(
+      File pubSpec, Version oldVersion, Version newVersion) async {
+    String newPubSpec = pubSpec.readAsStringSync().replaceFirst(
+        'version: "${oldVersion}"', "version: " + '"${newVersion.toString()}"');
```

Comment:
I don't understand how different quotation is processed. This line seems to only deal with double quotes. Is there a test for double/single/no quotes?